addons/website_sale_stock/static/src/js/variant_mixin.js

```diff
@@ -5,7 +5,6 @@ import publicWidget from "@web/legacy/js/public/public_widget";
 import { renderToFragment } from "@web/core/utils/render";
 import { formatFloat } from "@web/core/utils/numbers";
 
-import "@website_sale/js/website_sale";
```

Comment:
to be able to remove that, we should also make sure this file is imported before website_sale file defined in website_sale module otherwise we could still have the issue you try to fix. 🤔 

<https://github.com/odoo/odoo/blob/4b601a522b59f152766561151af0cf35378008ba/addons/website_sale_stock/__manifest__.py#L38>